torch/csrc/distributed/autograd/rpc_messages/rpc_with_autograd.cpp

```diff
@@ -90,37 +96,39 @@ std::unique_ptr<RpcWithAutograd> RpcWithAutograd::fromMessage(
 
   std::vector<torch::Tensor> tensors = message.tensors();
   int64_t messageId = message.id();
-  // Decode message type, autograd context id and autograd message id.
+  // Decode message type, autograd context id, autograd message id and worker
```

Comment:
Might be good to clarify that this worker_id is the id of the node that sent this RPC (i.e. `fromWorkerId`)? I feel like since we are including worker_ids in the rpc layer, confusion can result in whether in id is for a destination or sender node